manifest.json

```diff
@@ -58,6 +58,12 @@
       "content.js"
     ],
     "run_at": "document_end"
+  }, {
+    "matches": ["https://www.courtlistener.com/*"],
+    "js": [
+      "install_notifier.js"
+    ],
+    "run_at": "document_end"
```

Comment:
Use of `document_end` rather than the default `document_idle` opens you up to potential race conditions with scripts for the page that are still executing. Presumably you're aware of this and confident that this is the right choice, or prepared to adjust it if not. Cf. https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/content_scripts.

Reply:
No, actually, I assumed that `document_end` was the normal case, since it's what we normally use. Thanks, tweaked too. 

Jeesh. Small PR, but many tweaks.